src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -238,7 +244,16 @@ else if(privacyCommand.equals("make public")){
       conversationStore.updateConversation(conversation);
     }
     else if(purpose.equals("Adding users")){
-      //TODO parse json here
+
+      String[] userNameArray = stringToArray(request.getReader().readLine());
```

Comment:
You shouldn't need to parse JSON yourself, Gson should do that for you. Something like: https://stackoverflow.com/a/18421994/1768702.

Reply:
Oh I thought you had to have a specific class for to parse into. Didn't know you could it to an array. Thanks!